src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,55 @@
-/* eslint-disable */
-import React from 'react';
+import { useAppSelector } from '../../app/hooks';
+import { Filter } from '../../features/filter';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+export const TodoList = () => {
+  const todos = useAppSelector<Todo[]>(state => state.todos);
+  const filter = useAppSelector<Filter>(state => state.filter);
+
+  const filteredTodos = todos.filter(todo => {
+    const lowerCaseQuery = filter.query.toLowerCase();
+    const lowerCaseTitle = todo.title.toLowerCase();
+
+    switch (filter.status) {
+      case 'active':
+        return !todo.completed && lowerCaseTitle.includes(lowerCaseQuery);
+      case 'completed':
+        return todo.completed && lowerCaseTitle.includes(lowerCaseQuery);
+      case 'all':
+      default:
+        return lowerCaseTitle.includes(lowerCaseQuery);
```

Comment:
Move this checking to above
```suggestion
      case 'active':
        return !todo.completed;
        
      case 'completed':
        return todo.completed;

      default:
        return lowerCaseTitle.includes(lowerCaseQuery);
```